ConsoleApp1/list.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConsoleApp1
+{
+    class list
+    {
+        public virtual void Func()

```

Comment:
Nope. This is not a cycle with all models as I expect

Reply:
You need to add a generic list of Lines and use foreach(or for) to call your Fucn method